hardware_interface/include/hardware_interface/components/joint.hpp

```diff
@@ -172,15 +178,15 @@ class Joint
    * the interfaces list, the hardware can choose which values to set.
    *
    * \param state list of doubles with states of the hardware.
-   * \param interfaces list of interfaces on which states have to be provided.
+   * \param interfaces list of interface names to provide for.
    * \return return_type::INTERFACE_VALUE_SIZE_NOT_EQUAL if state and interfaces arguments do not
    * have the same length; return_type::INTERFACE_NOT_FOUND if one of provided interfaces is not
    * defined for the joint; return_type::OK otherwise.
    */
   HARDWARE_INTERFACE_EXPORT
-  virtual
   return_type set_state(
-    const std::vector<double> & state, const std::vector<std::string> & interfaces);
+    const std::vector<double> & state,
```

Comment:
also one line is uncrustify is not against it.

Reply:
yeah, uncrustify didn't like these changes